backend/requirements.txt

```diff
@@ -15,4 +15,5 @@ requests==2.32.2
 requests_aws4auth==1.1.1
 sqlalchemy==1.3.24
 starlette==0.36.3
-alembic==1.13.1
\ No newline at end of file
+alembic==1.13.1
+openpyxl==3.1.5
```

Comment:
I agree on using `openpyxl` :) Just curious, did you choose it because it does not require MS Excel to be installed?

Reply:
Yes! We want to convert the CSV data to XLSX programatically. Therefore does not require MS Excel interface.
Therefore `openpyxl` was chosen.
There are different library which support the same operations and has good performance as compared to `openpyxl` but `openpyxl` provides the stable library while maintain the good performance